src/basic/filesystems.c

```diff
@@ -47,26 +57,50 @@ const FilesystemSet filesystem_sets[_FILESYSTEM_SET_MAX] = {
                 "cgroup\0"
                 "cgroup2\0"
                 "devpts\0"
+                "devtmpfs\0"
                 "mqueue\0"
                 "proc\0"
                 "sysfs\0"
         },
+        [FILESYSTEM_SET_ANONYMOUS] = {
+                .name = "@anonymous",
+                .help = "Anonymous inodes",
+                .value =
+                "anon_inodefs\0"
+                "pipefs\0"
+                "sockfs\0"
+        },
+        [FILESYSTEM_SET_APPLICATION] = {
+                .name = "@application",
+                .help = "Application virtual filesystems",
+                .value =
+                "autofs\0"
+                "fuse\0"
+                "overlay\0"
+        },
         [FILESYSTEM_SET_AUXILIARY_API] = {
                 .name = "@auxiliary-api",
                 .help = "Auxiliary filesystem API",
                 .value =
+                "binfmt_misc\0"
                 "configfs\0"
                 "efivarfs\0"
                 "fusectl\0"
                 "hugetlbfs\0"
+                "rpc_pipefs\0"
                 "securityfs\0"
         },
         [FILESYSTEM_SET_COMMON_BLOCK] = {
                 .name = "@common-block",
                 .help = "Common block device filesystems",
                 .value =
                 "btrfs\0"
+                "erofs\0"
+                "exfat\0"
```

Comment:
f2fs (😿), ntfs (non-fuse from recent new kernel)?

Reply:
send a follow-up pr?

is f2fs that common? never heard of it?

did the new ntfs driver take over the old ntfs driver's magic and name?

Reply:
added.